.chloggen/receiver-otlpjson-token-attr.yaml

```diff
@@ -0,0 +1,27 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: bug_fix
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: otlpjsonfilereceiver
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Parse token.attributes and append it to the log record
```

Comment:
What's token.attributes?

Reply:
https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/pkg/stanza/fileconsumer/emit/emit.go#L14

Reply:
I know :) Can you please make it more explicit for folks reading the changelog who might not know what that is?

Reply:
oh okay ;)